browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap

```diff
@@ -795,7 +795,7 @@ exports[`CopyNumberVariantPage with dataset gnomad_cnv_r4 with variant of type D
               onClick={[Function]}
               type="button"
             >
-              Population
+              Genetic ancestry group
```

Comment:
Can probably keep lower case

Reply:
Seems like the other column headers are title case so going with that for now, happy to do sentence case instead if you prefer.

Thanks for catching the case consistency. 🙏